### ISSUE TITLE
Comparison with standard model

### ISSUE THREAD

**Issue by a7b23** [itayhubara/BinaryNet#7]
Any suggestions on how to compare the training time with a non-binary MLP of same complexity? i.e. maybe a set of opt flags that I need to turn off to convert the Main_BinaryNet_MNIST.py into a standard MLP model?
Also, I'm getting a test accuracy of only ~95%(~10 epochs) on MNIST. What maximum accuracy can I expect on full training? How many epochs will be needed?

**Comment by renjiushuihan** [comment]
Well, I have just run the code of mnist.py(The theano version, not this one).  Here is the output of the last epoch:
`  LR:                            3.02775865823e-07
  training loss:                 0.00297216895865
  validation loss:               0.010455210574
  validation error rate:         0.999999979511%
  best epoch:                    980
  best validation error rate:    0.939999978989%
  test loss:                     0.00903553832935
  test error rate:               0.929999979213%
`
It's about 25-30 seconds/epoch on a TITAN GPU.
Maybe I will compare the training time with other models
